cmd/templ/generatecmd/watcher/watch.go

```diff
@@ -60,10 +68,12 @@ func shouldIncludeFile(name string) bool {
 }
 
 type RecursiveWatcher struct {
-	ctx    context.Context
-	w      *fsnotify.Watcher
-	Events chan fsnotify.Event
-	Errors chan error
+	ctx     context.Context
+	w       *fsnotify.Watcher
+	Events  chan fsnotify.Event
+	Errors  chan error
+	timers  map[string]*time.Timer
+	timerMu sync.Mutex
```

Comment:
I know the official go codebase places mutexes directly above the struct item they're protecting, so in this case you'd place the timerMu directly before timers. Not sure if that style is replicated here, but thought I'd mention it.

Reply:
Didn't know of this, but will take your word on it :)